### PR TITLE
[workaround] Fixes for deployed-network-environment.j2.yaml

### PR DESCRIPTION
**Pull request by stuggi**
Until the following two landed in ooo add this via the tarball:
- Fix net_cidr_map in environments/deployed-network-environment.j2.yaml
  https://review.opendev.org/c/openstack/tripleo-heat-templates/+/852257

- Set host_routes in environments/deployed-network-environment.j2.yaml
  https://review.opendev.org/c/openstack/tripleo-heat-templates/+/852258

### CODE REVIEW

**Review by abays — APPROVED** [review]
/lgtm

**Comment by olliewalsh** [comment]
> Until the following two landed in ooo add this via the tarball:
> 
> * Fix net_cidr_map in environments/deployed-network-environment.j2.yaml
>   https://review.opendev.org/c/openstack/tripleo-heat-templates/+/852257
> * Set host_routes in environments/deployed-network-environment.j2.yaml
>   https://review.opendev.org/c/openstack/tripleo-heat-templates/+/852258

File: ansible/files/osp/17.0/environments/deployed-network-environment.j2.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          - tripleo_vip={{ network.vip }}
+        subnets:
+        {%- for subnet in network.subnets %}
+          {{ subnet }}:
+            cidr: {{ network.subnets[subnet].ip_subnet }}
+            dns_nameservers: []
+            gateway_ip: {{ network.subnets[subnet].gateway_ip }}
+            host_routes:
+              {%- if network.subnets[subnet].routes %}
+              {%- for route in network.subnets[subnet].routes %}
```
Comment (olliewalsh) [inline ansible/files/osp/17.0/environments/deployed-network-environment.j2.yaml:44]: Worth making the change that Harald suggested in https://review.opendev.org/c/openstack/tripleo-heat-templates/+/852258?

Comment (stuggi) [reply ansible/files/osp/17.0/environments/deployed-network-environment.j2.yaml]: > Worth making the change that Harald suggested in https://review.opendev.org/c/openstack/tripleo-heat-templates/+/852258?

done

**Pull request merged**